### ISSUE TITLE
Publication location

### ISSUE THREAD

**Issue by dougmeredith** [Reedyuk/jsonrpc-kotlin-client#3]
Any plans to publish this to Maven Central or some other open reposiitory? It looks like you've built something quite interesting, but the personal access token requirement for accessing Github in the build is a barrier to entry.

**Comment by Reedyuk** [comment]
Hey Doug, yes it's a possibility, I have other open source projects on maven central so wouldn't be hard to copy the scripts.
Leave it with me and will update this issue when it's done

**Comment by Reedyuk** [comment]
For now you could just publish to maven local and use it in your project

**Comment by Reedyuk** [comment]
@dougmeredith - the repository and packages are public in github so you dont need a token.
```
repositories {
    maven("https://maven.pkg.github.com/Reedyuk/jsonrpc-kotlin-client")
}
```

**Comment by Reedyuk** [comment]
In this repository, it is consuming the library with no credentials:
https://github.com/Reedyuk/web3kotlin/commit/1f139a1f29a099036715e9d26f556d57f78c8320

**Comment by dougmeredith** [comment]
Thanks for looking into this. Also, I did not know about publishing to Maven local. Good tip!

**Comment by Reedyuk** [comment]
@dougmeredith good news, i have made a change and have published version 0.0.7 to sonatype, it should be in maven central in the next 24hrs.

**Comment by dougmeredith** [comment]
Much appreciated!
